stream-chat-android-client/api/stream-chat-android-client.api

```diff
@@ -2865,10 +2872,6 @@ public abstract interface class io/getstream/chat/android/client/uploader/FileUp
 	public abstract fun sendImage (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/io/File;Lio/getstream/chat/android/client/utils/ProgressCallback;)Lio/getstream/chat/android/client/utils/Result;
 }
 
-public final class io/getstream/chat/android/client/uploader/ProgressTrackerKt {
-	public static final fun toProgressCallback (Lio/getstream/chat/android/client/uploader/ProgressTracker;)Lio/getstream/chat/android/client/utils/ProgressCallback;
```

Comment:
This is also a breaking change. I'm not sure if we can avoid it but definitely, we should mention it in the CHANGELOG and provide a migration guide